pilot/pkg/bootstrap/server.go

```diff
@@ -1224,7 +1226,49 @@ func (s *Server) initMeshHandlers() {
 	})
 }
 
-func (s *Server) initWorkloadTrustBundle() {
+func (s *Server) addIstioCAToTrustBundle(args *PilotArgs) {
+	// TODO: Will need to rework once CA code is refactored to update certs dynamically
+
+	signingKeyFile := path.Join(LocalCertDir.Get(), "ca-key.pem")
+	if _, err := os.Stat(signingKeyFile); err != nil {
+		// If NOT self signed certificates
+		rootCertFile := path.Join(LocalCertDir.Get(), "root-cert.pem")
```

Comment:
this is the same logic used in CA code to detect plugin certs. Note plugin certs may be mounted in external K8s cluster and fetched and stored in this local directory.